validator/sawtooth_validator/journal/chain.py

```diff
@@ -747,6 +764,7 @@ def on_block_validated(self, commit_new_block, result):
         """
         try:
             with self._lock:
+                self._blocks_considered_count.inc()
```

Comment:
If this is the number of blocks that passed validation, consider "block_validated_count".  But really it seems like we should have metrics on the resulting status instead.

Reply:
This is the number of blocks that have gone through validation but are not necessarily valid.